src/display/text_layer.js

```diff
@@ -226,7 +530,61 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
+    expandTextDivs: function TextLayer_expandTextDivs(expandDivs) {
+      if (!this._enhanceTextSelection || !this._renderingDone) {
+        return;
+      }
+      if (!this.expanded) {
+        expand(this.bounds, this._viewport);
+        this.expanded = true;
```

Comment:
Good question, I'll look into that. The bounds shouldn't be needed again once the new values have been calculated.